build.gradle

```diff
@@ -11,5 +11,7 @@ repositories {
 
 dependencies {
     compile 'org.apache.commons:commons-lang3:3.4'
+    compile group: 'com.google.guava', name: 'guava', version: '11.0.2'
     testCompile group: 'junit', name: 'junit', version: '4.11'
+
```

Comment:
done, but forgot to merge the previous changes..the junit version should be 5.x.x